api/src/main/java/com/capitalone/dashboard/rest/FeatureBranchController.java

```diff
@@ -0,0 +1,50 @@
+package com.capitalone.dashboard.rest;
+
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import com.capitalone.dashboard.service.FeatureBranchService;
+import com.capitalone.dashboard.model.FeatureBranch;
+
+import java.util.List;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+
+@RestController
+public class FeatureBranchController {
+    
+    private final FeatureBranchService featureBranchService;
+
+    @Autowired
+    public FeatureBranchController(FeatureBranchService featureBranchService) {
+        this.featureBranchService = featureBranchService;
+    }
+
+    @RequestMapping(value = "/featurebranchbytimeframe/{timestamp1}/{timestamp2}", method = GET, produces = APPLICATION_JSON_VALUE)
```

Comment:
Given that we are exposing feature branches and then filtering by timeframe, would it make sense to have the following syntax?

```
/feature_branches?start_timestamp=1234&end_timestamp=9876
```